src/css/styles/_helpers.scss

```diff
@@ -35,33 +35,15 @@
   padding-left: 0.3em;
 }
 
-.h-current-lang {
-  color: $col-txt-gray-light !important;
-}
-
-.l-container-padding {
-  padding-right: 4rem;
-  padding-left: 4rem;
-}
-
-.c-section {
-  padding-bottom: 4rem;
-}
-
-.l-last-element {
-  margin-bottom: 3rem;
-}
-
-.l-center-y {
-  position: absolute;
-  top: 50%;
-  transform: translateY(-50%);
+.h-btn-divider {
+  padding: 0.6rem;
+  margin: 0.6rem 1.2rem 0.6rem 0;
+  font-family: $font-praktika-bold;
+  font-size: 0.6rem;
+  text-transform: uppercase;
+  letter-spacing: 0.1em;
 }
 
-.l-col-clean {
-  padding: 0;
-}
-
-.l-col-padding {
-  padding: 0.5rem;
+.h-current-lang {
+  color: $col-txt-gray-light !important;
 }
```

Comment:
`.h-border-bottom--orange` and `.h-border-bottom--thin-gray` are used only once. Could we move them where they belong? 

- [ ] `c-btn--underline-orange` vs `.h-border-bottom--orange`
- [ ] `c-btn--underline-thin-gray` vs `.h-border-bottom--thin-gray`
- [ ] not sure: the same with `.h-border-bottom` (`.c-btn--black`)
- [ ] not sure: the same with `.h-current-lang` (`.c-lang--current-lang`)

Reply:
Note:
`.h-border-bottom` changed to `.c-btn--underline`